packages/web-components/src/components/va-segmented-progress-bar/va-segmented-progress-bar.tsx

```diff
@@ -0,0 +1,73 @@
+import { Component, Event, EventEmitter, Host, Prop, h } from '@stencil/core';
+
+@Component({
+    tag: 'va-segmented-progress-bar',
+    styleUrl: 'va-segmented-progress-bar.css',
+    shadow: true,
+})
+export class VaSegmentedProgressBar {
+    /**
+     * Whether or not an analytics event will be fired.
+     */
+    @Prop() enableAnalytics: boolean;
+
+    /**
+     * The current segment in progress
+     */
+    @Prop() current: number;
+
+    /**
+     * The total number of segments in the progress bar
+     */
+    @Prop() total: number;
+
+    /**
+    * An override for the default aria label.
+    */
+    @Prop() label: string = '';
```

Comment:
Does this need to default to an empty string? Can we instead set the `Step ${this.current} of ${this.total}` here?

Reply:
<img width="606" alt="Screen Shot 2021-12-09 at 10 54 26 AM" src="https://user-images.githubusercontent.com/11822533/145430042-b322c11d-1dcc-40cb-b7f5-6f741451a9e6.png">
Sadly we cannot going to follow what Andres did in the Progress Bar PR #227